doc/how_to_contribute/how_to_contribute.rst

```diff
@@ -0,0 +1,11 @@
+How-To Contribute
+=================
+
+These how-to guides outline steps for contributing to MoveIt and this website.
```

Comment:
If folks from outside PickNik are going to be able to see this, there should probably be additional clarification here on what exactly this section is and a link to where they might actually want to go (user guides). This is also where people who are trying to submit PRs are probably going to land, so a section for how to contribute in that way may also be appropriate.

Reply:
I agree this needs information for people contributing to moveit itself.  Maybe at first we could just make this section for people contributing to the website.

I do like the idea of starting this with, "If you are looking for how-to guides for using moveit, go here."  or something similar.